store/src/main/java/org/apache/rocketmq/store/schedule/ScheduleMessageService.java

```diff
@@ -58,14 +59,16 @@
     private final ConcurrentMap<Integer /* level */, Long/* offset */> offsetTable =
         new ConcurrentHashMap<Integer, Long>(32);
     private final DefaultMessageStore defaultMessageStore;
+    private final BrokerStatsManager brokerStatsManager;
     private final AtomicBoolean started = new AtomicBoolean(false);
     private Timer timer;
     private MessageStore writeMessageStore;
     private int maxDelayLevel;
 
-    public ScheduleMessageService(final DefaultMessageStore defaultMessageStore) {
+    public ScheduleMessageService(final DefaultMessageStore defaultMessageStore, final BrokerStatsManager brokerStatsManager) {
```

Comment:
Add another constructor that support this feature enable system have a better compatibility.

Reply:
agree. it is better to keep the old constructor as default.

Reply:
done~~